examples/miral-shell/miral-app.sh

```diff
@@ -80,8 +80,14 @@ then
   # miral-shell can launch it's own terminal with Ctrl-Alt-T
   MIR_SERVER_FILE=${socket} WAYLAND_DISPLAY=${wayland_display} MIR_SERVER_ENABLE_X11=1 MIR_SERVER_SHELL_TERMINAL_EMULATOR=${terminal} NO_AT_BRIDGE=1 ${hostsocket} exec dbus-run-session -- ${bindir}${miral_server} ${enable_mirclient} $*
 else
+  x11_display_file=$(tempfile)
   # miral-kiosk (and mir_demo_server) need a terminal launched
-  MIR_SERVER_FILE=${socket} WAYLAND_DISPLAY=${wayland_display} ${hostsocket} ${bindir}${miral_server} ${enable_mirclient} $*&
+  if [ "${miral_server}" == "mir_demo_server" ]
+  then
+    MIR_SERVER_FILE=${socket} MIR_SERVER_ENABLE_X11=1 WAYLAND_DISPLAY=${wayland_display} ${hostsocket} ${bindir}${miral_server} ${enable_mirclient} $* --x11-displayfd 5 5>${x11_display_file}&
```

Comment:
It may be non-obvious to some what's going on, and that `x11_display_file` is a text file not a socket. A comment here might help.